pkgs/games/r2modman/default.nix

```
@@ -0,0 +1,81 @@
+{ lib
+, mkYarnPackage
+, fetchFromGitHub
+, gitUpdater
+, makeWrapper
+, makeDesktopItem
+, copyDesktopItems
+, electron
+}:
+
+let
+  execName = "r2modman";
+in
+mkYarnPackage rec {
+  name = "r2modman";
```

I always forget about this myself :stuck_out_tongue: 
```suggestion
  pname = "r2modman";
```

---

I thought this was the other way around now: `pname` is derived from `name` + `version`.

---

Nope, I had it backwards. #103997